answers.md

```diff
@@ -0,0 +1,60 @@
+#### Chapter 13 Self-Test Answers
+
+1) **D**.  It's _always_ 'All of the above'!
+
+2)  No, but the _Type_ _Wrappers_ can.
+
+3)
+```java
+public class FlightSched<T, V> {
+}
+```
+
+4)
+```java
+public class FlightSched<T, V extends Thread> {
+}
+```
+
+5)
+```java
+public class FlightSched<T, V extends T> {
+}
+```
+
+6) *?* is a **wildcard**.  It is used when the exact type is unknown and matches all valid types.
+
+7)  Yes.
+```java
+static void doSomething( SomeValue<? extends Number> ) {
+}
+```
+
+8)
+```java
+public static <T> T MyGen( T t) {
+  return t;

```

Comment:
yay! Thanks for the 'gentleman octopus'.  Disappointed to miss the meeting this morning.